src/main/java/pl/beata/zarzadzanie_2/TaskController.java

```diff
@@ -0,0 +1,46 @@
+package pl.beata.zarzadzanie_2;
+
+import java.time.LocalDate;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+public class TaskController {
+
+    @Autowired
+    TaskService taskService;
+
+    @GetMapping("/")
+    public String home(Model model) {
+        model.addAttribute("newTask", new Duty());
+        model.addAttribute("categories", Category.values());
```

Comment:
To można zostąpić w szablonie poprzez wywłanie SpEL `${T(pl.beata.zarzadzanie_2.Category).values()}`